core-bundle/src/File/Metadata.php

```diff
@@ -118,4 +133,28 @@ public function empty(): bool
     {
         return empty($this->values);
     }
+
+    public function getJsonLd(string $type = null): array
+    {
+        if (null === $type) {
+            return $this->jsonLd;
+        }
+
+        return $this->jsonLd[$type] ?? [];
+    }
+
+    public static function extractBasicJsonLd(self $metadata): array
```

Comment:
I made this `public static` on purpose so that if you want to access this somewhere else, you can. We can also make this `private` though.